src/openforms/forms/statistics.py

```diff
@@ -35,6 +35,11 @@ def export_registration_statistics(
     :arg limit_to_forms: A queryset of forms to limit the export to. If not provided or
       ``None`` is given, all forms are included.
     """
+    title_mappings = {
+        logevent.REGISTRATION_SUCCESS_EVENT: "Successfully registered",
+        logevent.FORM_SUBMIT_SUCCESS_EVENT: "Successfully completed",
+    }
```

Comment:
this is not properly picked up by translations, you'll have to create the full translatable string in the mapping.

Reply:
I thought it would be picked up since I have `format`..I will update it yes